src/joinRates/JoinRates.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useMemo, useState, useCallback } from 'react'
+
+import JoinRatesTable from './JoinRatesTable'
+import { JoinRatesProps } from './model'
+import { Quality } from '../quality/Quality'
+
+function getTableNames(tableNames?: Array<string>): { 'Table names': Array<string> } | {} {
+  return tableNames ? { 'Table names': tableNames } : {}
```

Comment:
In this case we will have an object with the next properties:
` key: 'someId',
 id: 'someId',
 'Table names': ['table1', 'table2']`
 
 Why don't we call this property 'table_names'? Do we have any reason for this?

Reply:
Yes, https://github.com/stoys-io/stoys-ui/pull/5/files#diff-b84bcd890798ea5eef8af508b710d81515ce00c2393554d023ddee9dafd2cca2R42 - it transforms keys into column names